ui/src/src/launches/LaunchAttributeStatsChart.js

```diff
@@ -27,12 +27,12 @@ class LaunchAttributeStatsChart extends SubComponent {
 
         this.state.data = props.stats;
         this.state.attrKey = props.attrKey;
-        this.setUpStatusSeries();
-        this.setState(this.state);
     }
 
     componentDidMount() {
         super.componentDidMount();
+        this.setState(this.state);
```

Comment:
You can omit super call here.
`this.setState` causes an extra re-render, which is a bad practice from the performance standpoint. It's better to get rid of this call.

Reply:
fixed